BTVN10/main.js

```diff
@@ -0,0 +1,167 @@
+//b1:in bang nhan
```

Comment:
Nếu ghi thì ghi như thế này, chú ý các dấu cách và cái nào viết hoa
```suggestion
// B1: in bang nhan
```
Nhưng, khi viết/đặt tên thì mặc định dùng tiếng Anh, không được dùng ngôn ngữ khác, ngoại trừ được yêu cầu, hoặc bắt buộc phải dùng ngôn ngữ khác